TimeService/work/timeservice/implem/default/C/src/timeservice.h

```diff
@@ -0,0 +1,38 @@
+/* Header file for function TimeService
+ * Generated by TASTE (kazoo/templates/skeletons/c-header/function.tmplt)
+ * DO NOT EDIT THIS FILE, IT WILL BE OVERWRITTEN DURING THE BUILD
+ */
+
+#pragma once
+
+#include "dataview-uniq.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+
+void timeservice_startup(void);
+
+/* Provided interfaces */
+void timeservice_PI_CfsTimestampCmp( const asn1SccCfsTimestamp *, const asn1SccCfsTimestamp *, asn1SccComparisonResult * );
+
+
+void timeservice_PI_ClockStatus( asn1SccClockStatusEnum * );
+
+
+void timeservice_PI_CucTimestamp( const asn1SccULongInteger *, asn1SccCucTimestamp * );
```

Comment:
This is not in the requirements, but how can we get a Cfs timestamp?

Reply:
Please include a symmetric interface for cfs timestamp

Reply:
Added interface and conversion